node_cli/index.js

```diff
@@ -0,0 +1,59 @@
+const inquirer = require("inquirer");
+const chalk = require("chalk");
+const clear = require("clear");
+const figlet = require("figlet");
+const realmAuth = require("./realm/realmAuth");
+const main = require("./main");
+
+const output = (text, type) => {
+  if (type == "result") console.log(chalk.cyanBright(text));
+  if (type == "header") console.log(chalk.cyanBright.bold("\n" + text + "\n"));
+  if (type == "error") console.log(chalk.red.bold("\n ❗" + text + " ❗\n"));
+};
+
+let authedUser;
+
+clear();
+console.log(
+  chalk.green.bold(
+    figlet.textSync("realm-cli", {
+      font: "colossal",
+    })
+  )
+);
+
+function run() {
+  output("*** WELCOME ***", "header");
+  output("Please log in to your Realm account:", "header");
+  inquirer
+    .prompt([
+      {
+        type: "input",
+        name: "email",
+        message: "Email:",
+      },
+      {
+        type: "password",
+        name: "password",
+        message: "Password:",
+        mask: "*",
+      },
+    ])
+    .then((input) => {
+      realmAuth.login(input.email, input.password).then((result) => {
+        authedUser = result;
+        if (!authedUser.isLoggedIn) {
+          output("Something went wrong logging you in.", "error");
+          return;
```

Comment:
Not sure when this particular error would actually happen. If the login promise resolves to a user it means that the user successfully logged in. If something went wrong it should be handled as an error in the login function.

```js
try {
  const user = await realmAuth.login(input.email, input.password)
} catch (error) {
  output("Something went wrong logging you in.", "error");
}
```